Missing files in prepackaged source builds for Linux

Compilation works almost fine on Ubuntu until this point (also the file /libtransmission/subprocess-test.cmd is missing from the nightly builds, I had to manually insert it)

```
Scanning dependencies of target transmission-gtk-po
[ 54%] Generating transmission-gtk-an.mo
[ 54%] Generating transmission-gtk-ar.mo
[ 54%] Generating transmission-gtk-ast.mo
[ 55%] Generating transmission-gtk-az.mo
[ 55%] Generating transmission-gtk-be.mo
[ 55%] Generating transmission-gtk-bg.mo
[ 56%] Generating transmission-gtk-bn.mo
[ 56%] Generating transmission-gtk-bo.mo
[ 57%] Generating transmission-gtk-br.mo
[ 57%] Generating transmission-gtk-bs.mo
[ 57%] Generating transmission-gtk-ca.mo
[ 58%] Generating transmission-gtk-ca@valencia.mo
[ 58%] Generating transmission-gtk-ceb.mo
[ 58%] Generating transmission-gtk-ckb.mo
[ 59%] Generating transmission-gtk-cs.mo
[ 59%] Generating transmission-gtk-cy.mo
[ 59%] Generating transmission-gtk-da.mo
[ 60%] Generating transmission-gtk-de.mo
[ 60%] Generating transmission-gtk-el.mo
[ 61%] Generating transmission-gtk-en_AU.mo
[ 61%] Generating transmission-gtk-en_CA.mo
[ 61%] Generating transmission-gtk-en_GB.mo
[ 62%] Generating transmission-gtk-eo.mo
[ 62%] Generating transmission-gtk-es.mo
[ 62%] Generating transmission-gtk-et.mo
[ 63%] Generating transmission-gtk-eu.mo
[ 63%] Generating transmission-gtk-fa.mo
[ 63%] Generating transmission-gtk-fi.mo
[ 64%] Generating transmission-gtk-fil.mo
[ 64%] Generating transmission-gtk-fo.mo
[ 65%] Generating transmission-gtk-fr.mo
[ 65%] Generating transmission-gtk-fr_CA.mo
[ 65%] Generating transmission-gtk-ga.mo
[ 66%] Generating transmission-gtk-gl.mo
[ 66%] Generating transmission-gtk-gv.mo
[ 66%] Generating transmission-gtk-he.mo
[ 67%] Generating transmission-gtk-hi.mo
[ 67%] Generating transmission-gtk-hr.mo
[ 67%] Generating transmission-gtk-hu.mo
[ 68%] Generating transmission-gtk-hy.mo
[ 68%] Generating transmission-gtk-ia.mo
[ 69%] Generating transmission-gtk-id.mo
[ 69%] Generating transmission-gtk-is.mo
[ 69%] Generating transmission-gtk-it.mo
[ 70%] Generating transmission-gtk-ja.mo
[ 70%] Generating transmission-gtk-jbo.mo
[ 70%] Generating transmission-gtk-ka.mo
[ 71%] Generating transmission-gtk-kk.mo
[ 71%] Generating transmission-gtk-ko.mo
[ 71%] Generating transmission-gtk-ku.mo
[ 72%] Generating transmission-gtk-ky.mo
[ 72%] Generating transmission-gtk-li.mo
[ 73%] Generating transmission-gtk-lt.mo
[ 73%] Generating transmission-gtk-lv.mo
[ 73%] Generating transmission-gtk-mk.mo
[ 74%] Generating transmission-gtk-ml.mo
[ 74%] Generating transmission-gtk-mr.mo
[ 74%] Generating transmission-gtk-ms.mo
[ 75%] Generating transmission-gtk-mt.mo
[ 75%] Generating transmission-gtk-my.mo
[ 75%] Generating transmission-gtk-nb.mo
[ 76%] Generating transmission-gtk-nds.mo
[ 76%] Generating transmission-gtk-nl.mo
[ 77%] Generating transmission-gtk-nn.mo
[ 77%] Generating transmission-gtk-oc.mo
[ 77%] Generating transmission-gtk-pa.mo
[ 78%] Generating transmission-gtk-pl.mo
[ 78%] Generating transmission-gtk-pt.mo
[ 78%] Generating transmission-gtk-pt_BR.mo
make[2]: *** No rule to make target '../po/pt_PT.po', needed by 'po/transmission-gtk-pt_PT.mo'.  Stop.
CMakeFiles/Makefile2:1250: recipe for target 'po/CMakeFiles/transmission-gtk-po.dir/all' failed
make[1]: *** [po/CMakeFiles/transmission-gtk-po.dir/all] Error 2
Makefile:162: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/libtransmission/Makefile.am
+++ b/libtransmission/Makefile.am
@@ -316,3 +316,6 @@ watchdir_generic_test_CPPFLAGS = -DWATCHDIR_TEST_FORCE_GENERIC $(AM_CPPFLAGS)
 rename_test_SOURCES = rename-test.c $(TEST_SOURCES)
 rename_test_LDADD = ${apps_ldadd}
 rename_test_LDFLAGS = ${apps_ldflags}
+
+EXTRA_DIST = \
+  subprocess-test.cmd

--- a/macosx/Makefile.am
+++ b/macosx/Makefile.am
@@ -95,6 +95,8 @@ EXTRA_DIST = \
     InfoTabButtonBack.m \
     InfoTabButtonCell.h \
     InfoTabButtonCell.m \
+    InfoTabMatrix.h \
+    InfoTabMatrix.m \
     InfoTextField.h \
     InfoTextField.m \
     InfoActivityViewController.h \

--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -68,6 +68,7 @@ pa
 pl
 pt
 pt_BR
+pt_PT
 ro
 ru
 si

